src/app/meshes/components/MeshDetails.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <MeshCharts :mesh-insight="meshInsight" />
+      </template>
+    </KCard>
+
+    <KCard>
+      <template #body>
+        <div class="columns">
+          <DefinitionList>
+            <DefinitionListItem
+              v-for="(value, property) in basicMesh"
+              :key="property"
+              :term="t(`http.api.property.${property}`)"
+            >
+              <KBadge
+                v-if="typeof value === 'boolean'"
+                :appearance="value ? 'success' : 'danger'"
+              >
+                {{ value ? 'Enabled' : 'Disabled' }}
+              </KBadge>
+
+              <template v-else-if="property === 'name' && typeof value === 'string'">
+                <TextWithCopyButton :text="value" />
+              </template>
+
+              <template v-else>
+                {{ value }}
+              </template>
+            </DefinitionListItem>
+          </DefinitionList>
+
+          <DefinitionList>
+            <DefinitionListItem
+              v-for="(value, property) in extendedMesh"
+              :key="property"
+              :term="t(`http.api.property.${property}`)"
+            >
+              <KBadge
+                v-if="typeof value === 'boolean'"
+                :appearance="value ? 'success' : 'danger'"
+              >
+                {{ value ? 'Enabled' : 'Disabled' }}
+              </KBadge>
+
+              <template v-else>
+                {{ value }}
+              </template>
+            </DefinitionListItem>
+          </DefinitionList>
+
+          <DefinitionList>
+            <DefinitionListItem :term="`Policies (${totalPolicyCount})`">
+              <ul>
+                <li
+                  v-for="(policyType, index) in policyTypes"
+                  :key="index"
+                >
+                  <router-link
+                    :to="{
+                      name: 'policies-list-view',
+                      params: {
+                        policyPath: policyType.path
+                      }
+                    }"
+                  >
+                    {{ policyType.name }}: {{ policyType.total }}
+                  </router-link>
+                </li>
+              </ul>
+            </DefinitionListItem>
+          </DefinitionList>
+        </div>
+      </template>
+    </KCard>
+
+    <KCard>
+      <template #body>
+        <ResourceCodeBlock
+          id="code-block-mesh"
+          :resource-fetcher="fetchMesh"
+          :resource-fetcher-watch-key="mesh?.name || null"
+        />
```

Comment:
github syntax highlighting turned blue here, it's probably their bug but maybe we can just do `>` like on line 68 with `router-link` so it just works.

Reply:
The syntax highlighting GitHub uses is unfortunately quite out-of-date for .vue files.

Somewhat modern syntax like `mesh?.name` trips up the syntax highlighter. We _could_ "fix" this by wrapping such expressions in parenthesis which has proven to work (and we do it sometimes), but really, we shouldn’t alter our code to satisfy the GH diff highlighting.

And yeah, the reason is not the self-closing syntax on line 85.